src/main/resources/db/migration/V134__ViabilityTestsCut.sql

```diff
@@ -0,0 +1,55 @@
+ALTER TABLE viability_tests ADD COLUMN seeds_compromised INTEGER;
+ALTER TABLE viability_tests ADD COLUMN seeds_empty INTEGER;
+ALTER TABLE viability_tests ADD COLUMN seeds_filled INTEGER;
+
+INSERT
+INTO viability_tests (accession_id,
+                      remaining_grams,
+                      remaining_quantity,
+                      remaining_units_id,
+                      seeds_compromised,
+                      seeds_empty,
+                      seeds_filled,
+                      seeds_sown,
+                      test_type)
+SELECT id,
+       remaining_grams,
+       remaining_quantity,
+       remaining_units_id,
+       cut_test_seeds_compromised,
+       cut_test_seeds_empty,
+       cut_test_seeds_filled,
+       COALESCE(cut_test_seeds_compromised, 0)
+           + COALESCE(cut_test_seeds_empty, 0)
+           + COALESCE(cut_test_seeds_filled, 0),
+       3 -- Cut
```

Comment:
is this the equivalent of a limit 3?

Reply:
Nope, but clearly my comment is too terse. 3 is the test type ID for cut tests, and `--` is the SQL comment delimiter. I'll make this clearer.